src/service/jqlService.ts

```diff
@@ -0,0 +1,25 @@
+import JqlModel from "../model/jqlModel";
+/**
+ * Created by JJax on 19.11.2016.
+ */
+
+const MAX_RESULTS = -1;
+
+export default class JqlService {
+    public doRequest(jqlModel: JqlModel, httpClient, callback) {
+        httpClient.post({url: "/rest/api/2/search", json: this.prepareRequestBody(jqlModel)},
```

Comment:
Spoko w TODO'sie poprzednio dałem, że to musi być zrefaktoryzowane do promisów albo asynca, także wiedziałem o problemie. Niemniej done.